`netcomp` not working properly

Assuming have copied [config](https://imperialcollegelondon.github.io/SWMManywhere/config_guide/#minimum-viable-configuration) and updated accordingly. @dalonsoa it'd be amazing if you could take a look please.

```bash
python -m venv ./sa_env
./sa_env/Scripts/activate.bat
pip install swmmanywhere
python -m swmmanywhere --config_path=minimum_viable_template.yml 

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\bdobson\Downloads\temp\sa_env\Lib\site-packages\swmmanywhere\__main__.py", line 8, in <module>
    from swmmanywhere import swmmanywhere
  File "C:\Users\bdobson\Downloads\temp\sa_env\Lib\site-packages\swmmanywhere\swmmanywhere.py", line 22, in <module>
    from swmmanywhere.metric_utilities import iterate_metrics, validate_metric_list
  File "C:\Users\bdobson\Downloads\temp\sa_env\Lib\site-packages\swmmanywhere\metric_utilities.py", line 21, in <module>
    import netcomp
ModuleNotFoundError: No module named 'netcomp'
```

@dalonsoa new bug with the distribution:

```
Run if [[ "false" != 'true' ]]; then
/tmp/baipp/dist/swmmanywhere-0.1.2-py3-none-any.whl: W009: Wheel contains multiple toplevel library entries:
  netcomp/
  swmmanywhere/
Error: Process completed with exit code 1.
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -89,7 +89,8 @@ Source = "https://github.com/ImperialCollegeLondon/SWMManywhere"
 packages = [ "src" ]
 
 [tool.hatch.build.targets.wheel]
-packages = [ "src" ]
+only-include = ["src"]
+sources = ["src"]
 
 [tool.hatch.version]
 source = "vcs"

